src/main.ts

```diff
@@ -64,6 +65,61 @@ async function main() {
 		console.log(`Logged in as ${client.user?.tag}`);
 	});
 
+	client.on('messageReactionAdd', async (reaction, user) => {
+		// When a reaction is received, check if the structure is partial
+		if (reaction.partial) {
+			// If the message this reaction belongs to was removed, the fetching might result in an API error which should be handled
+			try {
+				await reaction.fetch();
+			} catch (error) {
+				log.error(client, {message: 'Something went wrong when fetching the message:',stack:'',name:'React Message Error'});
```

Comment:
This ignores the config setting for logging errors, I think logging to console.error is fine for this